net/openfortivpn/files/openfortivpn-ppp-up

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+PPP_IPPARAM="$6"
+
+. /lib/netifd/netifd-proto.sh
+proto_init_update "$IFNAME" 1 1
+proto_set_keep 1
+[ -n "$PPP_IPPARAM" ] && {
+	[ -n "$IPLOCAL" ] && proto_add_ipv4_address "$IPLOCAL" 32 "" "${IPREMOTE:-2.2.2.2}"
```

Comment:
Why do we set the IP to 2.2.2.2 if $IPREMOTE is not set?

Reply:
I'm not sure. It does seem weird. I copied this from the ppp-up script. The only change was on line 9.

Reply:
That is the commit that adds this address.
https://github.com/openwrt/openwrt/commit/5caa45d0b396972b313fb3d99908ba978b684570#diff-d7672a29ff6fe8d75e5edc9d5b7e95a7
The maintainer is not active anymore. So maybe @dedeckeh could help use hier?

Reply:
Openfortivpn defaults to 192.0.2.1 which is an RFC 3330 "TEST-NET" ip address. Perhaps that would be a good choice here?